docs/api/dt/countna.rst

```diff
@@ -0,0 +1,71 @@
+
+.. xfunction:: datatable.countna
+    :src: src/core/expr/head_reduce_unary.cc op_countna
+    :tests: tests/test-reduce.py
+    :cvar: doc_dt_countna
+    :signature: countna(cols)
+
+
+    Count the number of NA values for each column from `cols`.
+
+    Parameters
+    ----------
+    cols: Expr
+        Input columns.
+
+    return: Expr
+        f-expression having one row, and the same names and number of columns
+        as in `cols`. All the returned column stypes are `int64`.
+
+    except: TypeError
+        The exception is raised when one of the columns from `cols`
+        has a non-numeric and non-string type.
+
+
+
+    Examples
+    --------
+
+    .. code-block:: python
+
+        >>> from datatable import dt, f
+        >>>
+        >>> df = dt.Frame({'A': [1, 1, 2, None, 1, 2],
+        ...                'B': [None, 2, 3, 4, None, 5],
+        ...                'C': [1, 2, 1, 1, 2, 4]})
+        >>> df
+           |     A      B      C
+           | int32  int32  int32
+        -- + -----  -----  -----
+         0 |     1     NA      1
+         1 |     1      2      2
+         2 |     2      3      1
+         3 |    NA      4      1
+         4 |     1     NA      2
+         5 |     2      5      4
+        [6 rows x 3 columns]
+
+    Get the countna of all rows:
```

Comment:
*columns

Also, it's probably better to say "count of NAs" instead of "countna" here.

Reply:
Ack